gossip/src/crds.rs

```diff
@@ -669,6 +686,18 @@ impl CrdsDataStats {
                 self.votes.put(slot, num_nodes + 1);
             }
         }
+
+        if should_report_message_signature(entry.value.signature) {
+            datapoint_info!(
+                "cluster_info_crds_message_signatures",
+                ("crds_origin", entry.value.pubkey().to_string(), String),
```

Comment:
Can we also limit to first 8 bytes of pubkeys?

Reply:
so i was thinking about this, yes we can for sure. was just thinking it may be easier to query on the backend if we have the full string pubkey string. But i guess the convenience of having the whole string is not worth the overhead in metrics.